app/models/businessdetails/ActivityStartDate.scala

```diff
@@ -30,7 +30,7 @@ object ActivityStartDate {
 
   implicit val formRule: Rule[UrlFormEncoded, ActivityStartDate] = From[UrlFormEncoded] { __ =>
     import jto.validation.forms.Rules._
-      (__ \ "startDate").read(newAllowedPastAndFutureDateRule) map ActivityStartDate.apply
+      (__ \ "startDate").read(newAllowedPastAndFutureDateRule("error.required.date", "error.invalid.date.after.1700", "error.invalid.date.past")) map ActivityStartDate.apply
```

Comment:
Why here date after 1700? Was this ticket to update messages in business details section?

Reply:
No but the method that I refactored for agent trading premises was also being used in Business details. Thus adding the messages as parameters in the model. 1700 date should be on this question as it is in trading premises